modules/light-clients/08-wasm/types/wasm.go

```diff
@@ -1,61 +1,59 @@
 package types
 
 import (
-	"bytes"
-	"slices"
+	"cosmossdk.io/collections"
 
-	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/ibcwasm"
 )
 
-// GetCodeHashes returns all the code hashes stored.
-func GetCodeHashes(ctx sdk.Context, cdc codec.BinaryCodec) (CodeHashes, error) {
-	wasmStoreKey := ibcwasm.GetWasmStoreKey()
-	store := ctx.KVStore(wasmStoreKey)
-	bz := store.Get([]byte(KeyCodeHashes))
-	if len(bz) == 0 {
-		return CodeHashes{}, nil
-	}
-
-	var hashes CodeHashes
-	err := cdc.Unmarshal(bz, &hashes)
-	if err != nil {
-		return CodeHashes{}, err
-	}
+// GetCodeHasheKeySet returns the KeySet collection for the code hashes.
+func GetCodeHashKeySet() collections.KeySet[[]byte] {
+	wasmStoreService := ibcwasm.GetWasmStoreService()
+	sb := collections.NewSchemaBuilder(wasmStoreService)
 
-	return hashes, nil
+	return collections.NewKeySet(sb, CodeHashesKey, "code_hashes", collections.BytesKey)
```

Comment:
maybe we should set the map name as a constant somewhere?

Reply:
Could do, but this is the standard used in cosmos sdk modules. So I'd wanna be consistent with them.